File: postit_wall/nextjs_space/fix_caps.mjs
import { PrismaClient } from '@prisma/client'

const prisma = new PrismaClient()

async function main() {
  const cats = await prisma.category.findMany()
  console.log("Categories:")
  for (const cat of cats) {
    if (cat.name === cat.name.toUpperCase() && cat.name.match(/[A-Z]/)) {
      console.log(`Uppercase found: ${cat.name} (ID: ${cat.id})`)
    }
  }
}

main().catch(console.error).finally(() => prisma.$disconnect())

File: postit_wall/nextjs_space/fix_caps.py
from prisma import Client
import asyncio

async def main():
    client = Client()
    await client.connect()
    
    cats = await client.category.find_many()
    print("Found categories:")
    for cat in cats:
        if cat.name.isupper():
            print(f"Uppercase found: {cat.name} (ID: {cat.id})")

if __name__ == '__main__':
    asyncio.run(main())
